plugins/logitech-hidpp/README.md

```diff
@@ -74,6 +75,17 @@ the bootloader and runtime modes are treated as the same device.
 The Bolt receiver enumerates as a hidraw device both in runtime and
 bootloader mode, but with different HIDRAW devIDs.
 
+Peripherals paired to a receiver are enumerated as separate hidraw
+devices but those device files can't be used for DFU. Instead, all the
+DFU-related messages need to be piped through the hidraw device file of
+the receiver. They are polled and queried by the receiver and listed as
+its children. Note that this will likely change once the Logitech Linux
+driver supports Bolt devices.
```

Comment:
hopefully not meaning that there is a regression coming down the road!

Reply:
Fingers crossed...